source/GameAction.cpp

```diff
@@ -189,6 +190,13 @@ void GameAction::LoadSingle(const DataNode &child)
 		if(maxDays < minDays)
 			swap(minDays, maxDays);
 		events[GameData::Events().Get(child.Token(1))] = make_pair(minDays, maxDays);
+	} else if (key == "play") {
+		if (hasValue) {
+			music = child.Token(1);
+			playMusic = true;
+		} else {
+			child.PrintTrace("Error: play doesn't have a specified file. If you want to stop music, do 'play \"\"'");
+		}
 	}
 	else if(key == "fail" && child.Size() >= 2)
```

Comment:
Unrelated to this PR, but I just noticed that this could use hasValue as well.
```suggestion
	else if(key == "fail" && hasValue)
```